### PR TITLE
Add Gateway quick links section

### PR DESCRIPTION
**Pull request by mheap**
### Summary
Adds a "Quick Links" launchpad on the Gateway index page

### Reason
To help people find what they need faster

### Testing
[/gateway/latest/](https://deploy-preview-4396--kongdocs.netlify.app/gateway/latest/)

### CODE REVIEW

File: src/gateway/index.md [hunk 1]
```diff
@@ -3,6 +3,39 @@ title: Kong Gateway
 subtitle: API gateway built for hybrid and multi-cloud, optimized for microservices and distributed architectures
 ---
 
+## Quick Links
+
+<div class="docs-grid-install">
+
+  <a href="#features" class="docs-grid-install-block no-description">
+    <img class="install-icon" src="/assets/images/icons/documentation/icn-flag.svg" alt="">
```
Comment (lena-larionova) [inline src/gateway/index.md:11]:
```suggestion
    <img class="install-icon no-image-expand" src="/assets/images/icons/documentation/icn-flag.svg" alt="">
```
Need to add a class so this doesn't happen:
Screen Shot 2022-09-06 at 10 36 14 AM

It should really go the other way - add a class when you need to expand, not have it be the default. But for now, this is what we've got.

File: src/gateway/index.md [hunk 2]
```diff
@@ -3,6 +3,39 @@ title: Kong Gateway
...
+    <div class="install-text">Features</div>
+  </a>
+
+  <a href="/gateway/{{page.kong_version}}/get-started/" class="docs-grid-install-block no-description">
+    <img class="install-icon" src="/assets/images/icons/documentation/icn-learning.svg" alt="">
```
Comment (lena-larionova) [inline src/gateway/index.md:16]:
```suggestion
    <img class="install-icon no-image-expand" src="/assets/images/icons/documentation/icn-learning.svg" alt="">
```

File: src/gateway/index.md [hunk 3]
```diff
@@ -3,6 +3,39 @@ title: Kong Gateway
...
+    <div class="install-text">Quick Start</div>
+  </a>
+
+  <a href="/gateway/{{page.kong_version}}/install/" class="docs-grid-install-block no-description">
+    <img class="install-icon" src="/assets/images/icons/documentation/icn-deployment-color.svg" alt="">
```
Comment (lena-larionova) [inline src/gateway/index.md:21]:
```suggestion
    <img class="install-icon no-image-expand" src="/assets/images/icons/documentation/icn-deployment-color.svg" alt="">
```

File: src/gateway/index.md [hunk 4]
```diff
@@ -3,6 +3,39 @@ title: Kong Gateway
...
+    <div class="install-text">Install</div>
+  </a>
+
+  <a href="/hub/" class="docs-grid-install-block no-description">
+    <img class="install-icon" src="/assets/images/icons/documentation/icn-api-plugins-color.svg" alt="">
```
Comment (lena-larionova) [inline src/gateway/index.md:26]:
```suggestion
    <img class="install-icon no-image-expand" src="/assets/images/icons/documentation/icn-api-plugins-color.svg" alt="">
```

File: src/gateway/index.md [hunk 5]
```diff
@@ -3,6 +3,39 @@ title: Kong Gateway
...
+    <div class="install-text">Plugins</div>
+  </a>
+
+  <a href="/gateway/{{page.kong_version}}/admin-api/" class="docs-grid-install-block no-description">
+    <img class="install-icon" src="/assets/images/icons/documentation/icn-admin-api-color.svg" alt="">
```
Comment (lena-larionova) [inline src/gateway/index.md:31]:
```suggestion
    <img class="install-icon no-image-expand" src="/assets/images/icons/documentation/icn-admin-api-color.svg" alt="">
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Page links work and the icon boxes look good!

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**